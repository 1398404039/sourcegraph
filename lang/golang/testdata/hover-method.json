[
  {
    "id": 0,
    "jsonrpc": "2.0",
    "method": "initialize",
    "params": {
      "rootPath": "testdata",
      "capabilities": {}
    }
  },
  {
    "id": 1,
    "jsonrpc": "2.0",
    "method": "textDocument/hover",
    "params": {
      "textDocument": {
        "uri": "gopath/src/mux/route.go"
      },
      "position": {
        "line": 199,
        "character": 27
      }
    }
  },
  {
    "id": 2,
    "jsonrpc": "2.0",
    "method": "shutdown",
    "params": null
  }
]
