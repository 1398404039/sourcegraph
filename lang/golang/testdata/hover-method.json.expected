[
  {
    "id": 0,
    "result": {
      "capabilities": {
        "hoverProvider": true,
        "definitionProvider": true,
        "referencesProvider": true,
        "workspaceSymbolProvider": true
      }
    },
    "jsonrpc": "2.0"
  },
  {
    "id": 1,
    "result": {
      "contents": [
        {
          "language": "go",
          "value": "Match"
        },
        {
          "language": "text/definfo",
          "value": "{\"URI\":\"file:///gopath/src/mux/route.go\",\"UnitType\":\"GoPackage\",\"Unit\":\"mux\",\"Path\":\"Match\"}"
        }
      ],
      "range": {
        "start": {
          "line": 199,
          "character": 23
        },
        "end": {
          "line": 199,
          "character": 27
        }
      }
    },
    "jsonrpc": "2.0"
  },
  {
    "id": 2,
    "result": null,
    "jsonrpc": "2.0"
  }
]